windows-gaming-driver/src/config.rs

```diff
@@ -47,19 +47,19 @@ pub struct MachineConfig {
     pub usb_devices: Vec<DeviceId>,
 }
 
-#[derive(Deserialize, Serialize, Debug, Clone)]
+#[derive(Deserialize, Serialize, Debug, Clone, Default)]
 pub struct StorageDevice {
     pub path: String,
     pub cache: String,
     pub format: String,
 }
 
-#[derive(Deserialize, Serialize, Debug, Clone)]
+#[derive(Deserialize, Serialize, Debug, Clone, Default)]
 pub struct NetworkConfig {
     pub bridges: Vec<String>, // TODO: custom usernet
 }
 
-#[derive(Deserialize, Serialize, Debug, Clone)]
+#[derive(Deserialize, Serialize, Debug, Clone, Default)]
```

Comment:
`StorageDevice`, `NetworkConfig` and `SambaConfig` shouldn't be `Default`. It doesn't make sense for any of them and it's not required either as they only appear in `Vec` and `Option`.